engine/src/mbliphonesensor.mm

```diff
@@ -294,6 +294,25 @@ static void initialize_core_location(void)
 
 ////////////////////////////////////////////////////////////////////////////////
 
+void MCSystemAllowBackgroundLocationUpdates(bool p_allow)
+{
+    NSDictionary *t_info_dict;
+    t_info_dict = [[NSBundle mainBundle] infoDictionary];
+    
+    NSArray *t_background_modes_array;
+    t_background_modes_array = [t_info_dict objectForKey: @"UIBackgroundModes"];
+    
+    BOOL t_plist_can_allow_background_location_updates = [t_background_modes_array containsObject: @"location"];
+    
+    if (t_plist_can_allow_background_location_updates)
+    {
+        if (p_allow)
+            s_location_manager.allowsBackgroundLocationUpdates = YES;
+        else
+            s_location_manager.allowsBackgroundLocationUpdates = NO;
```

Comment:
@livecodepanos This function needs a call to `initialize_core_location` before accessing members of `s_location_manager`